src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/Extensions/DocumentExtensions.cs

```diff
@@ -203,18 +203,5 @@ public static IEnumerable<Document> GetLinkedDocuments(this Document document)
                 yield return solution.GetRequiredDocument(linkedDocumentId);
             }
         }
-
-        public static bool CanAddImportsInHiddenRegions(this Document document)
-        {
-#if !CODE_STYLE
-            // Normally we don't allow generation into a hidden region in the file.  However, if we have a
-            // modern span mapper at our disposal, we do allow it as that host span mapper can handle mapping
-            // our edit to their domain appropriate.
-            var spanMapper = document.Services.GetService<ISpanMappingService>();
-            return spanMapper != null && spanMapper.SupportsMappingImportDirectives;
-#else
-            return false;
-#endif
```

Comment:
did this logic move elsewhere?

Reply:
Yes, moved to AddImportPlacementOptions record struct.